src/main/java/org/sunbird/user/registration/service/UserRegistrationServiceImpl.java

```diff
@@ -305,6 +306,38 @@ public void initiateCreateUserFlow(String registrationCode) {
 		}
 	}
 
+	public SBApiResponse getApprovedDomains() {
+		SBApiResponse response = createDefaultResponse(Constants.API_APPROVED_DOMAINS);
```

Comment:
I thought this method adds all the necessary information into the response... 

if not -- just use ProjectUtil.createDefaultResponse